examples/jenkins/jenkins-persistent-template.json

```diff
@@ -15,6 +15,9 @@
     }
   },
   "message": "A Jenkins service has been created in your project.  Log into Jenkins with your OpenShift account.  The tutorial at https://github.com/openshift/origin/blob/master/examples/jenkins/README.md contains more information about using this template.",
+  "labels": {
+      "app": "jenkins-persistent"
```

Comment:
The app label should be parameterized so that when a DB template is used with another application and we want to associate it with that app we can.